pyproject.toml

```diff
@@ -32,8 +32,7 @@ dependencies = [
   "fsspec>=2022.8.0",
   "numpy>=1.16",
   "PyYAML>=6.0",
-  "readlif>=0.6.4",
-  "xarray>=0.16.1",
+  "xarray>=0.16.1", "readlif @ https://github.com/Arcadia-Science/readlif.git",
```

Comment:
This formatting is a bit funky

Reply:
Sorry, I had used pixi CLI to add the dependency, and that's what it produced.

I would anyways update this draft PR and review the formatting once a new version of `readlif` is released, and depending on feedback here how to handle the (proposed) new test file.